src/application/TestSuite.ts

```diff
@@ -106,6 +107,13 @@ export class TestSuite {
 
     return pmOperations
   }
+
+  public getTestTypeFromContractTests = (type: string): ContractTestConfig => {
```

Comment:
this should return `ContractTestConfig | undefined` instead of casting to `ContractTestConfig`